corehq/apps/userreports/reports/builder/sources.py

```diff
@@ -0,0 +1,16 @@
+
+
+
+def get_source_type_from_report_config(report_config):
+    """
+    Get a report builder source type from an existing ReportConfiguration object
+    :param report_config:
+    :return:
+    """
+
+    return {
+        "CommCareCase": "case",
+        "XFormInstance": "form"
```

Comment:
Nit: Sometimes you use `DATA_SOURCE_TYPE_.*` and sometimes you use their values. I'm sure their values won't change, so I know this will be fine, but the pedant in me would use one or the other.

Reply:
thanks, and good catch. i fixed this and the other one I saw in this PR. agree the constants are kind of silly, though I think they add value mainly when trying to understand how something is used. i.e. not all instances of "case" in the codebase have the same semantic meaning.